counterpartylib/lib/backend/__init__.py

```diff
@@ -241,9 +241,12 @@ def pubkeyhash_to_pubkey(pubkeyhash, provided_pubkeys=None):
                 scriptsig = vin['scriptSig']
                 asm = scriptsig['asm'].split(' ')
                 if len(asm) >= 2:
-                    pubkey = asm[1]
-                    if pubkeyhash == script.pubkey_to_pubkeyhash(util.unhexlify(pubkey)):
-                        return pubkey
+                    try:
```

Comment:
can you provide a comment to signify the try block deals with differences from P2SH scriptsigs?